### DISCUSSION TITLE
Flowchart?

### DISCUSSION THREAD

**Discussion by E-Frame** [requarks/wiki#3778] [Help / Questions]
Heyhey, I wanted to know if anyone could tell me how to integrate or write a flowchart style in wiki.js? Similar to how the pages are visualized in one of the overview windows?

Or how I could have the flowchart visualiser as the "parent" page of a group of pages?

**Comment by davidljung** [comment]
The Kroki and Mermaid plugin for extending Markdown syntax support a lot of diagrams types, including flowcharts.
